src/components/Nav/NewCollectiveModal.vue

```diff
@@ -270,11 +278,19 @@ export default {
 			this.nameIsTooShort = true
 		},
 
+		setNameIsTooLong() {
+			this.nameIsTooLong = true
+		},
+
 		clearName() {
 			this.name = ''
 		},
 
 		advanceToMembers() {
+			if (this.nameIsTooLong) {
```

Comment:
This should no longer be needed, as advancing to the next step is impossible due to `nameIsInvalid` being true if `nameIsTooLong`.